tests/tests.rs

```diff
@@ -622,6 +616,24 @@ fn test_default_builder_type() {
         }
     }
 
-    let outer = Outer::builder().a(3).b(5).build().outer();
+    impl From<Inner> for Outer {
+        fn from(value: Inner) -> Self {
+            Self(value)
+        }
+    }
+
+    let outer = Outer::builder().a(3).b(5).build();
     assert_eq!(outer, Outer(Inner { a: 3, b: 5 }));
 }
+
+#[test]
+fn test_into_set_generic() {
+    #[derive(Debug, PartialEq, TypedBuilder)]
+    #[builder(into)]
+    struct SomeStruct<T: Default> {
+        #[builder(default)]
+        a: T,
+    }
+
+    let some: SomeStruct<bool> = SomeStruct::builder().build();
```

Comment:
Please add an `assert_eq` that verifies the value. This will also pacify the linter bot.

Reply:
Oh whoops, forgot that, thanks